lib/rules/no-await-sync-events.ts

```diff
@@ -32,11 +35,23 @@ export default createTestingLibraryRule<Options, MessageIds>({
       noAwaitSyncEvents:
         '`{{ name }}` is sync and does not need `await` operator',
     },
-    schema: [],
+    schema: [
+      {
+        type: 'object',
+        properties: {
+          eventModules: {
+            enum: VALID_EVENT_MODULES,
```

Comment:
@Belco90 Is it an `enum` or an array of enum values? 🤔

`schema` says enum, default values says array of enum values.

Reply:
If I understood it correctly, the enum allows to specify an array? Maybe I'm wrong and I ser the incorrect type.

Reply:
Tests are also using a value instead of array btw